Localization/de-DE/statistics.yaml

```
@@ -0,0 +1,45 @@
+---
+url: "https://foldingathome.org/statistics/"
+language: "de-DE"
+title: "STATISTIK"
+text: [
+# First two paragraphs
+#"One of the best ways to help Folding@home is by recruiting your friends and family. Start by sharing our project with them. Then join a team or even start your own team. The more points your team earns, the closer we come to finding cures.
+#On this page you will find access to statistics for individuals and teams who have joined together to earn points and compete with other teams. Some of us are quite intense in our approach to folding. We have team websites, we supe up our computers, and we drive the technology forward by reporting bugs and making suggestions about how to improve the software.",
+"Eine der besten Möglichkeiten, Folding@home zu helfen, besteht darin, Ihre Freunde und Familie zu rekrutieren. Teilen Sie zunächst unser Projekt mit ihnen. Schließen Sie sich dann einem Team an oder gründen Sie sogar ein eigenes Team. Je mehr Punkte Ihr Team verdient, desto näher kommen wir uns Heilmittel finden.
+Auf dieser Seite finden Sie Zugriff auf Statistiken für Personen und Teams, die sich zusammengeschlossen haben, um Punkte zu sammeln und mit anderen Teams zu konkurrieren. Einige von uns gehen sehr intensiv mit dem Falten um. Wir haben Team-Websites, wir verbessern unsere Computer und wir treiben die Technologie voran, indem wir Fehler melden und Vorschläge zur Verbesserung der Software machen. ",
+
+# Maximise your effort stylistic bold text
+#"MAXIMIZE your EFFORT",
+"MAXIMIERE deinen AUFWAND",
+
+# SET UP YOUR TEAM
+#"SET UP YOUR TEAM",
+"RICHTE HIER DEIN TEAM EIN",
+
+# Associated text for setting up a team
+#"If you are interested in setting up a team of your own, you can get started here.",
+"Wenn Sie daran interessiert sind, ein eigenes Team aufzubauen, können Sie hier loslegen.",
+
+# START A TEAM button
+#"START A TEAM",
+"STARTEN SIE EIN TEAM",
+
+# CHANGE YOUR TEAM INFO button
+#"CHANGE YOUR TEAM INFO",
+"ÄNDERN SIE IHRE TEAMINFO",
+
+# Lost team password? link
+#"Lost team password?",
+"Team Passwort verloren?",
+
+# Heading 1
+#"MONITORING STATS UPDATES AUTOMATICALLY: AVAILABLE DOWNLOADS AND RULES OF USE",
+"AUTOMATISCHE ÜBERWACHUNG STATS UPDATES: VERFÜGBARE DOWNLOADS UND NUTZUNGSBESTIMMUNGEN",
```

That first part sounds a bit clunky to me. Maybe more like this:
```suggestion
"STATISTIK-AKTUALISIERUNGEN AUTOMATISCH ÜBERWACHEN: VERFÜGBARE DOWNLOADS UND NUTZUNGSBESTIMMUNGEN",
```
Or like this if we accept "Update" as common enough:
```suggestion
"STATISTIK-UPDATES AUTOMATISCH ÜBERWACHEN: VERFÜGBARE DOWNLOADS UND NUTZUNGSBESTIMMUNGEN",
```